fetch.bs

```diff
@@ -7793,15 +7802,21 @@ method steps are:
    <li><p><a for=/>Resolve</a> <var>p</var> with <var>responseObject</var>.
   </ol>
 
+ <li><p>Let <var>serializedError</var> be the result of [$StructuredSerialize$]
```

Comment:
the result of calling. Not sure you can use [$] in the fetch spec

Reply:
`[$` should be fine, it's only used for abstract ops, but StructuredSerialize is indeed an abstract op.

Reply:
OK then!